packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "26.0.0",
+  "version": "27.0.0",
```

Comment:
updating core version because we will be creating a release after this PR is merged